src/components/PeerGroupsUpdate.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {useDispatch, useSelector} from "react-redux";
```

Comment:
Codacy has a fix for the issue: [Replace `useDispatch,·useSelector` with `·useDispatch,·useSelector·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
import { useDispatch, useSelector } from "react-redux";
```